README.md

```diff
@@ -38,6 +38,14 @@ A sample for Inky only, both host development and on-device is available as [pap
 
 A sample for using it with Scenic both for host development and on-device is available as [pappersverk/sample_scenic_inky](https://github.com/pappersverk/sample_scenic_inky).
 
+
+There are multiple variants of wHAT, pHAT, etc, and they are hard to distinguish.
+You can read the display information from your Inky's EEPROM by invoking `Inky.EEPROM.read/0`.
+
+```elixir
+Inky.EEPROM.read()
+```
```

Comment:
You probably should mention what one could do with this information below the example.